battle-service/src/main/java/by/ibondarau/tictactoe/battleservice/controller/BattleController.java

```diff
@@ -13,88 +23,100 @@
+        this.battleService = battleService;
+        this.modelMapper = modelMapper;
+    }
+
     @GetMapping("/{battleId}")
     public ResponseEntity<BattleResponseDto> get(@PathVariable Integer battleId) {
-        BattleResponseDto dto = new BattleResponseDto()
-                .id(battleId)
-                .firstPlayerId(1)
-                .secondPlayerId(2)
-                .size(3)
-                .status("Finished")
-                .moves(Collections.emptyList())
-                .nextMove(null)
-                .result("First wins")
-                .created(ZonedDateTime.now().toInstant())
-                .started(ZonedDateTime.now().toInstant())
-                .finished(ZonedDateTime.now().toInstant());
-        return ResponseEntity.ok(dto);
+        return ResponseEntity.ok(modelMapper.battleToBattleResponse(battleService.getBattle(battleId)));
     }
 
     @PostMapping
     public ResponseEntity<BattleResponseDto> createBattle(@RequestBody CreateBattleDto createBattleDto) {
-        BattleResponseDto dto = new BattleResponseDto()
-                .id(1)
-                .firstPlayerId(createBattleDto.userId())
-                .secondPlayerId(2)
-                .size(3)
-                .status("Created")
-                .moves(Collections.emptyList())
-                .nextMove(1)
-                .result(null)
-                .created(ZonedDateTime.now().toInstant())
-                .started(null)
-                .finished(null);
-        return ResponseEntity.ok(dto);
+        logger.info("BattleController.createBattle() - started");
+        logger.info("BattleController.createBattle() - createBattleDto=" + createBattleDto.toString());
+
+        Battle battle = battleService.createBattle(
+                createBattleDto.getUserId(),
+                createBattleDto.getSize(),
+                FirstMoveRule.valueOf(createBattleDto.getFirstMoveRule())
+        );
+
+        return ResponseEntity.ok(modelMapper.battleToBattleResponse(battle));
     }
 
     @PostMapping("/{battleId}/join")
     public ResponseEntity<BattleResponseDto> joinBattle(@PathVariable Integer battleId, @RequestBody JoinBattleDto joinBattleDto) {
-        BattleResponseDto dto = new BattleResponseDto()
-                .id(battleId)
-                .firstPlayerId(1)
-                .secondPlayerId(joinBattleDto.secondPlayerId())
-                .size(3)
-                .status("Started")
-                .moves(Collections.emptyList())
-                .nextMove(1)
-                .result(null)
-                .created(ZonedDateTime.now().toInstant())
-                .started(ZonedDateTime.now().toInstant())
-                .finished(null);
-        return ResponseEntity.ok(dto);
+        logger.info("BattleController.createBattle() - started");
+        logger.info("BattleController.createBattle() - battleId=" + battleId + ", playerId=" + joinBattleDto.getSecondPlayerId());
```

Comment:
use {} instead as params placeholders of string concatenation
ie
`log.info( "BattleController.createBattle() - battleId={} , playerId={}",battleId,joinBattleDto.getSecondPlayerId())`